### PR TITLE
Add code snippet for speechbrain audio classification

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: widgets/src/lib/interfaces/Libraries.ts [hunk 1]
```diff
@@ -101,6 +101,13 @@ model = Speech2Text.from_pretrained(
 speech, rate = soundfile.read("speech.wav")
 text, *_ = model(speech)`;
 
+const espnetAudioClassification = (model: ModelData) =>
```
Comment (mishig25) [inline widgets/src/lib/interfaces/Libraries.ts:104]: this has to be `const speechbrainAudioClassification`, unless speechbrain is related to espnet?

Comment (osanseviero) [reply widgets/src/lib/interfaces/Libraries.ts]: Done, I messed working between two libraries at same time.

File: widgets/src/lib/interfaces/Libraries.ts [hunk 2]
```diff
@@ -189,6 +189,13 @@ nlp = spacy.load("${nameWithoutNamespace(model.modelId)}")
 import ${nameWithoutNamespace(model.modelId)}
 nlp = ${nameWithoutNamespace(model.modelId)}.load()`;
 
+const speechbrainAudioClassification = (model: ModelData) =>
+`from speechbrain.pretrained import EncoderClassifier
+
+model = EncoderClassifier.from_hparams(
+  "${model.modelId}"
+)`;
+
```
Comment (mishig25) [inline widgets/src/lib/interfaces/Libraries.ts:198]: Nit: as in other speechbrain examples [here](https://github.com/huggingface/huggingface_hub/blob/b32ccaa4ab32d5b5ca40273d8a9f321332953de6/widgets/src/lib/interfaces/Libraries.ts#L199-L215),

does it need to follow the format:
```py
from speechbrain.pretrained import EncoderClassifier
audioclassification_model = EncoderClassifier.from_hparams("${model.modelId}") 
audioclassification_model.method("file.wav")
```

Comment (osanseviero) [reply widgets/src/lib/interfaces/Libraries.ts]: Done

**Review by mishig25 — APPROVED** [review]
lgtm!

**Pull request merged**